src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -130,12 +137,6 @@ export const BaseCourseSearchResults = ({
   }
   return (
```

Comment:
when I select all courses in the course selector stepper screen should I see an Enroll button also above the table to the right? I did not see it in the local version when I opened the first screen